src/webgl/p5.Camera.js

```diff
@@ -1617,11 +1617,6 @@ p5.Camera = class Camera {
     if (this.cameraType === 'default') {
       this._computeCameraDefaultSettings();
       this._setDefaultCamera();
-    } else {
```

Comment:
Currently framebuffers also have a duplicate version of this method: https://github.com/processing/p5.js/blob/53119bfc5c6f3f57c4791647d5d09d1977b24dc4/src/webgl/p5.Framebuffer.js#L55-L57

The duplication exists right now because it replaces `this._renderer.width`and height with `this.fbo.width` and height. If we remove the `else` branch here, I think we can just remove the overloaded `_resize` method from the framebuffer camera class, since the implementation would now be the same as for normal cameras.

Reply:
Sorry for the late response. This looks like it'd work perfectly. I'll remove the overloaded `_resize()`.